src/processor.ts

```diff
@@ -17,12 +17,16 @@ export const handler: Handler = Sentry.wrapHandler(async ({ records }) => ({
 
 const transformRecord = ({ recordId, data }: EventRecord): ResultRecord => {
   try {
-    const event = JSON.parse(Buffer.from(data, "base64").toString());
-    const { time, data: { raw } } = struct(event, OCSEvent); // prettier-ignore
+    const events = wrapList(JSON.parse(Buffer.from(data, "base64").toString()));
+    const {time} = struct(events[0], OCSEvent); // prettier-ignore
     const datetime = localFromISO(time);
```

Comment:
I know this isn't part of this review but is this function DST safe? It doesn't look like it.

Reply:
I'll add some tests to be sure, but this is going from a UTC datetime to a local time, which should be safe.

Reply:
oh I see, sorry.

Reply:
Better safe than sorry!